### PR TITLE
don't pretend we're testing interactive tiebreaking

### PR DESCRIPTION
**Pull request by artoonie**

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/TabulatorSession.java
Comment (HEdingfield) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java:1]: Confused about why this file is appearing in here? Didn't I make these changes already in a separate PR that was merged in?

Comment (artoonie) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java:1]: github diffs have been acting up all day on this repo, not sure why. I can fix this by merging develop into this branch.

**Review by HEdingfield — APPROVED** [review]
